server.js

```diff
@@ -1,33 +1,85 @@
-import express from 'express'
-import bodyParser from 'body-parser'
-import cors from 'cors'
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-// Defines the port the app will run on. Defaults to 8080, but can be 
+import express from 'express';
+import bodyParser from 'body-parser';
+import cors from 'cors';
+import netflixData from './data/netflix-titles.json';
+
+// Defines the port the app will run on. Defaults to 8080, but can be
 // overridden when starting the server. For example:
 //
 //   PORT=9000 npm start
-const port = process.env.PORT || 8080
-const app = express()
+console.log(netflixData.length);
+
+const port = process.env.PORT || 8080;
+const app = express();
 
 // Add middlewares to enable cors and json body parsing
-app.use(cors())
-app.use(bodyParser.json())
+app.use(cors());
+app.use(bodyParser.json());
 
 // Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
-})
+  res.send(
+    'paths: movies, movies/movie, genres/genre, actors/actor, directors/director, year/year, countries/country'
+  );
+});
+
+app.get('/movies', (req, res) => {
+  res.json(netflixData);
+});
+
+// route for getting data on a certain movie
+app.get('/movies/:movie', (req, res) => {
+  const movie = req.params.movie;
+  const chosenMovie = netflixData.filter(
+    (item) => item.title.toString().toLowerCase() === movie.toLowerCase()
+  );
+  res.json(chosenMovie);
+});
```

Comment:
Hmm, I think you've done kind of a mix here. Using `filter()` you get an array with the result back. If this route is going to return a single result, you could rename it `/movies/:id` and then use `find()` to find an id and to make sure you only get a single result.

You could keep the filter and move it to the `/movies` endpoint but use it as a query param. Path params are mostly used to identify a specific resource, so when you just want to filter, you use query params.

Another thing to think about is that when you're using === the param has to be an exact match with the title in your data, so if I just write "Irishman" it won't be a match to "The Irishman". To give the user the possibility to search for titles including a specific word you can use `includes()`.

```
const movie = req.query.movie;

const chosenMovie = netflixData.filter((item) =>
    item.title.toString().toLowerCase().includes(movie)
```

Then it would find "The Irishman", even if the user just searches for "Irishman".